ebean-test/testconfig/ebean-db2.properties

```diff
@@ -5,3 +5,4 @@ ebean.test.username=admin
 ebean.test.password=admin
 datasource.default=db2-11
 ebean.db2-11.databasePlatformName=db2luw
+ebean.lengthCheck=utf8
```

Comment:
This is something, that maybe shoud be moved to databasePlatform.
On the other side, we cannot know the exact charset, the DB2 database is using.

Reply:
I think let have it explicit per database first and see.